linkerd/app/core/src/profiles.rs

```diff
@@ -216,21 +213,14 @@ where
                         Ok(Async::NotReady) => return Ok(Async::NotReady),
                         Ok(Async::Ready(())) => {}
                         Err(err) => {
-                            error!(
-                                "profile service unexpected error (dst = {}): {:?}",
-                                self.dst, err,
-                            );
+                            error!("profile service unexpected error: {:?}", err,);
                             return Ok(Async::Ready(()));
                         }
                     };
-
-                    let req = api::GetDestination {
-                        path: self.dst.clone(),
-                        context_token: self.context_token.clone(),
-                        ..Default::default()
-                    };
-                    debug!("getting profile: {:?}", req);
-                    let rspf = self.service.get_profile(grpc::Request::new(req));
+                    debug!("getting profile");
```

Comment:
should we render the request is this message?

Reply:
the dst is already in the context now